src/__tests__/extensions/replay/sessionrecording.test.ts

```diff
@@ -825,6 +900,27 @@ describe('SessionRecording', () => {
             })
         })
 
+        describe('masking', () => {
+            it('passes remote masking options to rrweb', () => {
+                posthog.config.session_recording.maskAllInputs = undefined
+
+                posthog.persistence?.register({
+                    [SESSION_RECORDING_MASKING]: { maskAllInputs: true, maskTextSelector: '*' },
+                })
```

Comment:
nit: we should probably call onRemoteConfig here to avoid binding to the storage but 🤷